frontend/src/pages/create_campaign/index.tsx

```diff
@@ -170,7 +195,11 @@ const CreateCampaign = () => {
         navigate("/dashboard");
       })
       .catch(() => {
-        console.error("something went wrong");
+        console.error("Something went wrong");
+        pushMessage({
+          message: "Something went wrong on backend!",
+          type: "error",
+        });
```

Comment:
can we also try to get the json error message from the backend and show that? you can see an example of how we do that here: https://github.com/csesoc/chaos/blob/9c3d95b1862ff69924cf090f2e901bedd1796651/frontend/src/pages/auth_success/index.tsx#L31-L33
`string` type should be fine here, and can just do normal type annotation here instead of using `as`. `resp.json()` itself can also error tho if the response is invalid so maybe we should catch that as well lmao

Reply:
Pls double check I've done right <3

Reply:
mostly looks good aside from one small thing :)